### PR TITLE
Rename Workflow to Pipeline

### PR DESCRIPTION
**Pull request by recena**
The wiki page was created https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Remote+Loader+Plugin

@reviewbybees

### CODE REVIEW

**Comment by jtnord** [comment]
:bee:

**Comment by lordofthejars** [comment]
:bee:

**Comment by recena** [comment]
@lordofthejars @jtnord Thanks for your time

**Comment by recena** [comment]
@reviewbybees done

**Comment by lordofthejars** [comment]
You are welcome. A pleasure.

**Comment by recena** [comment]
@oleg-nenashev Are you the current maintainer?

File: pom.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
   <name>Jenkins Workflow Remote Loader Plugin</name>
```
Comment (oleg-nenashev) [inline pom.xml]: :bug: Missing change

File: pom.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
...
   <description>Allows loading Workflow groovy files from remote locations (enhanced version of the built-in load command)</description>
```
Comment (oleg-nenashev) [inline pom.xml]: :bug: Missing change

**Comment by recena** [comment]
@oleg-nenashev done

**Comment by oleg-nenashev** [comment]
:bee:

**Pull request merged**